pkgs/top-level/all-packages.nix

```diff
@@ -4691,7 +4693,7 @@ in
     '';
   });
 
-  pamtester = callPackage ../tools/security/pamtester { };
+  pantable = python3Packages.callPackage ../tools/text/pantable { };
```

Comment:
python packages should usually be in pkgs/top-level/python-packages.nix. Not sure why this package should be at top-level.